code/datums/gamemode/dynamic/dynamic_rulesets_latejoin.dm

```diff
@@ -68,7 +68,7 @@
 //        RAGIN' MAGES (LATEJOIN)           ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //                                          //
 //////////////////////////////////////////////1.01 - Lowered weight from 3 to 2
-
+/* //Currently disabled
```

Comment:
I recommend setting the weight for the rulesets to 0 so they can still be executed by admins.